collector/pkg/component/consumer/processor/aggregateprocessor/processor.go

```diff
@@ -174,6 +174,8 @@ func newNetRequestLabelSelectors() *aggregator.LabelSelectors {
 		aggregator.LabelSelector{Name: constlabels.ContentKey, VType: aggregator.StringType},
 		aggregator.LabelSelector{Name: constlabels.DnsDomain, VType: aggregator.StringType},
 		aggregator.LabelSelector{Name: constlabels.KafkaTopic, VType: aggregator.StringType},
+		aggregator.LabelSelector{Name: constlabels.RocketMQRequestCode, VType: aggregator.IntType},
+		aggregator.LabelSelector{Name: constlabels.RocketMQResponseCode, VType: aggregator.IntType},
```

Comment:
Please add RocketMQTopic into LabelSelectors since you will use it as the request_content in Entity Metric.
All the labels of aggregated metrics should be kept during aggregation.

Reply:
thanks, i will add